src/styles/index.scss

```diff
@@ -1,3 +1,9 @@
 body {
   margin: 0;
+  width: 1024px;
```

Comment:
It is generally not recommended to set a fixed width on the body element as it can cause layout issues on different screen sizes. Consider using a responsive approach, such as setting a max-width and using percentages or viewport units.